app/javascript/shared/components/tags.jsx

```diff
@@ -258,11 +258,11 @@ class Tags extends Component {
       }
     } else if (
       (e.keyCode < 65 || e.keyCode > 90) &&
-      e.keyCode !== KEYS.COMMA &&
-      e.keyCode !== KEYS.DELETE &&
-      e.keyCode !== KEYS.LEFT &&
-      e.keyCode !== KEYS.RIGHT &&
-      e.keyCode !== KEYS.TAB
+      e.keyCode != KEYS.COMMA &&
```

Comment:
Any reason the `!==` doesn't work? They're both numbers, so coalescing the values doesn't seem to be necessary.

https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/shared/components/tags.jsx#L4

```javascript
const KEYS = {
  UP: 38,
  DOWN: 40,
  LEFT: 37,
  RIGHT: 39,
  TAB: 9,
  RETURN: 13,
  COMMA: 188,
  DELETE: 8,
};
```

Reply:
I originally changed `!=` to `!==` because of the eslint failure messages, but when I originally submitted the PR one of the checks was failing because it was `!==` and not `!=`. It was the codeclimate check that was failing. 

This is my first PR and seeing a required check fail spooked me a bit, so I changed the `!==` back to `!=` to get the check passing. I wasn't sure which one y'all would want because the two errors are contradicting one another. I could easily make the change y'all want to be implemented.

Reply:
No worries. I was just peeking in some PRs. Awesome that this is your first PR. Don't be spooked 👻 by the failures.

Reply:
Hi @nickytonline, interesting! In general I would respect eslint over codeclimate too. Let us know if you remember what was the issue :D

Reply:
@rhymes the eslint message said something in regards to `!==` being considered good practice because it uses the type-safe equality operators.